logserver/jwt.php

```diff
@@ -0,0 +1,74 @@
+<?php
+// SPDX-FileCopyrightText: 2009 Fermi Research Alliance, LLC
+// SPDX-License-Identifier: Apache-2.0
+
+/*  Script to test JWT generation and verification in PHP:
+    php jwt.php // To run self test
+    php jwt.php TOKEN  // To decode the token
+    php jwt.php PAYLOAD TOKEN  // To encode PAYLOAD and save it to TOKEN
+    Uses the configuration in logging_config.json:
+    - secret_key_path (/var/lib/gwms-factory/server-credentials/jwt_secret.key)
+    - secret_key (VG9rZVNlY3JldEtleQo=)
```

Comment:
Is it okay/safe to have this secret key spelled out here?

Reply:
removed